dockerfiles/attestation-service/Dockerfile

```diff
@@ -10,6 +10,8 @@ COPY patches/ patches/
 COPY packages/utils/package.json packages/utils/
 COPY packages/typescript/package.json packages/typescript/
 COPY packages/dev-utils/package.json packages/dev-utils/
+COPY packages/protocol/package.json packages/protocol/
```

Comment:
Do we have to add these back?

Reply:
I don't think we have to, but I wanted to avoid publishing contractkit with this change given its urgency. Happy to make a follow-on PR once the the changes are published